code/game/machinery/rechargestation.dm

```diff
@@ -308,6 +308,11 @@
 					if(istype(O,/obj/item/weapon/melee/defibrillator))
 						var/obj/item/weapon/melee/defibrillator/D = O
 						D.charges = initial(D.charges)
+					if(istype(O,/obj/item/weapon/reagent_containers/syringe))
```

Comment:
Fix it, it'll just keep getting longer this way (which I figured it would a year ago in the quoted post)